source/_integrations/vacuum.xiaomi_miio.markdown

```diff
@@ -18,8 +18,9 @@ Currently supported services are:
 - `return_to_base`
 - `locate`
 - `clean_spot`
-- `set_fan_speed`
-- remote control of your robot.
+- `set_fan_speed` 
+   (On Xiaomi app version 5.6.34 changed from -> to; Quiet -> Silent, Balanced -> Standard, Turbo -> Medium, Max -> Turbo)
```

Comment:
This is not related to the app version, but the Home Assistant version. I think it would make more sense to just list the valid values. The changelog/breaking changes on the release notes are for helping current people.

```suggestion
  Fan speeds: `Silent`, `Standard`, `Medium`, `Turbo`.
```